### DISCUSSION TITLE
How to pass structured clone algorithm

### DISCUSSION THREAD

**Discussion by linonetwo** [ReactiveX/rxjs#6139] [Q&A] [answered]
Recently I found Observable can't pass the structured clone algorithm, for example in electron preload script you create an Observable, and want to pass it to the Renderer process, then the pipe and subscribe method on the Observable will become undefined: https://github.com/electron/electron/issues/28176

I'm going to find a way to allow pass Observable across `contextBridge.exposeInMainWorld` which use structured clone algorithm that wipes out all method from Observable.

**Comment by kwonoj** [comment, marked as answer]
You have answer already in posted link. It is known constraint of structured clone.
